docker/install-scripts/base.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 set -e
 
-apk update
-apk add git curl openssl
-apk add py-pip
-apk add --virtual=build gcc libffi-dev musl-dev openssl-dev python3-dev make
+apt update
```

Comment:
This is an expensive operation. I think the calling Dockerfile should do this and the scripts should just assume it is done.